web/pandas/config.yml

```diff
@@ -51,6 +51,8 @@ navbar:
       target: /community/ecosystem.html
   - name: "Contribute"
     target: /contribute.html
+  - name: "Donate"
+    taget: /donate.html
```

Comment:
we need to direct this to the right place eg our numfocus account 

any idea how to do that?

Reply:
This is already doing so, the donate page is linked to NF